apps/system/test/marionette/lib/system.js

```diff
@@ -389,6 +389,17 @@ System.prototype = {
     return frame;
   },
 
+  closeBrowser: function(url) {
```

Comment:
What is the url used for? I think this method should be named 'closeActiveApp' (or closeAppByUrl, if we use the parameter)